files/src/main/java/com/afollestad/materialdialogs/files/util/ContextExt.kt

```diff
@@ -0,0 +1,23 @@
+/**
+ * Designed and developed by Aidan Follestad (@afollestad)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.afollestad.materialdialogs.files.util
+
+import android.content.Context
+import java.io.File
+
+fun Context.getExternalFilesDir(): File? {
```

Comment:
This should be `internal` or marked with `@RestrictTo(LIBRARY)`

Reply:
You are right, thank you. I will add `internal` to keep consistency with other Extensions files.